eZ/Publish/Core/MVC/Symfony/Templating/Twig/Extension/QueryRenderingExtension.php

```diff
@@ -57,9 +62,15 @@ function (string $type, string $renderer, array $options): ?string {
 
     private function createControllerReference(array $options): ControllerReference
     {
-        return new ControllerReference('ez_query_render::renderQuery', [
-            'options' => $options,
-        ]);
+        $pageParam = isset($options['pagination']['page_param']) ? $options['pagination']['page_param'] : 'page';
```

Comment:
FYI : The default value for `page_param` is also hardcoded here : https://github.com/ezsystems/ezplatform-kernel/blob/v1.3.17/eZ/Publish/Core/MVC/Symfony/Controller/QueryRenderController.php#L76-L81